### ISSUE TITLE
error TS7030: Not all code paths return a value

### ISSUE THREAD

**Issue by lderks** [cstefanache/angular2-img-cropper#70]
Hi,

I'm getting this error:

```node_modules/ng2-img-cropper/src/imageCropper.ts(841,12): error TS7030: Not all code paths return a value.```

**Comment by lderks** [comment]
This is a TSLint semantic error. Could you please update that line of code so it will pass the TSlint validation?

**Comment by lderks** [comment]
To keep TSLint satisfied, change it to the code below.

```
public getDragTouchForID(id: any) {
        let currentDragTouch;
		for (let i = 0; i < this.currentDragTouches.length; i++) {
            if (id === this.currentDragTouches[i].id) {
                currentDragTouch = this.currentDragTouches[i];
            }
        }
		return currentDragTouch;
    }
```

**Comment by cstefanache** [comment]
fixed and will be released today with 0.7
